nsca_client/README.rst

```diff
@@ -40,14 +40,24 @@ To configure this module, you need to:
 
 * Code the methods which will be called by the NSCA checks.
 
-Such methods must return a tuple ``(RC, MESSAGE)`` where ``RC`` is an integer,
-and ``MESSAGE`` a unicode string. ``RC`` values and the corresponding status are:
+Such methods must return a tuple ``(RC, MESSAGE, PERFORMANCE_DATA)`` where ``RC`` is an integer,
+``MESSAGE`` a unicode string AND ``PERFOMANCE_DATA`` is a dictionary.
```

Comment:
Typo: `PERFORMANCE_DATA` (a `R` is missing)

Reply:
After reading the code, I saw that performance data is not mandatory (which is a good thing), but the README suggests the opposite. Can you reformulate the description?